docs-2.0/9.about-license/2.license-management-suite/3.license-manager.md

```diff
@@ -0,0 +1,200 @@
+# License Manager
+
+License Manager (LM) is a service that runs on a server for you to manage licenses. You can use an LM client that communicates with the LM service to load [license keys](2.license-center.md) and view license information, including the license validity period and purchased nodes. By configuring the LM service in the NebulaGraph database and its associated software, the validity of the license can be verified to ensure the normal use of the NebulaGraph database and its associated software.
```

Comment:
manage licenses => manage license. One lm only map to one license. it's a 1v1 relationship.

Reply:
fix content => By configuring the LM service address in the NebulaGraph....